app/controls/drag-drop.js

```diff
@@ -1,10 +1,7 @@
-import API from 'ipfs-api'
 import notifier from 'node-notifier'
 import {join} from 'path'
 import {getIPFS, logger} from './../init'
```

Comment:
The whole code base already uses es6 modules as everything gets transpiled anyway. If they are different repos we can have that differentiation, but in this case I'd rather have consistency in the repo. 

(Also this is not changed in this PR)